codecov.yml

```diff
@@ -0,0 +1,22 @@
+codecov:
+  require_ci_to_pass: no
+  max_report_age: off
+
+comment: false
+
+ignore:
+  - 'act/tests/*'
+  - 'act/*version*py'
+  - 'setup.py'
+  - 'versioneer.py'
+
+coverage:
+  precision: 2
+  round: down
+  status:
+    project:
+      default:
+        target: 95
```

Comment:
```suggestion
        target: 95
        threshold: 2%
```

Reply:
Ah perfect so there is a threshold! Changing now

Reply:
You betcha!

Reply:
Has been changed